content/docs/gke/deploy/deploy-cli.md

```diff
@@ -91,7 +91,7 @@ Follow these steps to deploy Kubeflow:
      The value of this variable cannot be greater than 25 characters. It must
      contain just the directory name, not the full path to the directory.
      The content of this directory is described in the next section.
-   * **${PROJECT}** - the _name_ of the GCP project where you want Kubeflow 
+   * **${PROJECT}** - the _PROJECT_ID_ of the GCP project where you want Kubeflow 
```

Comment:
Change `_PROJECT_ID_` to plain language:  project ID

I have two reasons for recommending the above change:

* There's a rendering problem for `_PROJECT_ID_` because the Markdown renderer doesn't know how to handle three underscores. See preview: https://deploy-preview-813--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-cli/

* Also, the page hasn't defined an environment variable named `PROJECT_ID`. It's therefore confusing to use syntax that looks like an env variable.

Reply:
changed this to project ID.